vite.config.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable import/no-extraneous-dependencies, import/no-anonymous-default-export, import/no-unused-modules */
+/* eslint-disable import/no-extraneous-dependencies, import/no-anonymous-default-export, import/no-unused-modules, jsdoc/require-file-overview */
```

Comment:
It's nice for normal JS linting to apply in these files, but some of the plugin rules don't make sense here, so I'm disabling those in this and the `vite.docs.js` files.